src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/User/Register.html

```diff
@@ -50,7 +50,7 @@
         </span>
         <div class="register-check">
             <input type="checkbox" name="registerCheck" data-ng-model="input.registerCheck" required="required">
-            {{ "TR__I_ACCEPT_TERMS_AND_CONDITIONS" | translate }}
+            {{ "TR__I_ACCEPT_THE" | translate }} <a href="{{ termsUrl }}" target="_blank">{{ "TR__TERMS_AND_CONDITIONS" | translate }}</a>
```

Comment:
I thought it's better than putting html in the translation string, which then needs to be marked as `safe` and thus allow translaters to break the site. I don't have a better idea how to do this at the moment.